L1Trigger/Phase2L1ParticleFlow/interface/TauNNId.h

```diff
@@ -0,0 +1,30 @@
+#ifndef L1TRIGGER_PHASE2L1PARTICLEFLOWS_TAUNNID_H
+#define L1TRIGGER_PHASE2L1PARTICLEFLOWS_TAUNNID_H
+
+#include <string>
+#include "PhysicsTools/TensorFlow/interface/TensorFlow.h"
+#include "DataFormats/L1TParticleFlow/interface/PFCandidate.h"
+
+class TauNNId {
+public:
+  TauNNId();
+  ~TauNNId();
+
+  void initialize(const std::string &iName, const std::string &iWeightFile, int iNParticles);
+  void SetNNVectorVar();
+  float EvaluateNN();
+  float compute(const l1t::PFCandidate &iSeed, l1t::PFCandidateCollection &iParts);
+
+  std::string fInput_;
```

Comment:
class member variables should be protected or private (unless there's a strong reason otherwise)

Reply:
done